From 3a44068b15ff9397a4ae8b2c8ad5f6b1583a73d9 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Mon, 27 Jul 2015 09:20:00 +0300
Subject: [PATCH 0266/1240] defconfig: Enable memtest in A8K and A38x
 defconfigs

	This command is usefull for testing memory stability under
	U-Boot.

Change-Id: I463e9b7db33f010eca0e60c57f9b501289fc63d9
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/22184
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
---
 configs/mvebu_armada38x_defconfig | 5 +++--
 configs/mvebu_armada8k_defconfig  | 1 +
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/configs/mvebu_armada38x_defconfig b/configs/mvebu_armada38x_defconfig
index 09e244f..82dc69e 100644
--- a/configs/mvebu_armada38x_defconfig
+++ b/configs/mvebu_armada38x_defconfig
@@ -1,19 +1,20 @@
 CONFIG_SPL=y
 +S:CONFIG_ARM=y
 +S:CONFIG_TARGET_ARMADA_38X=y
-+S:CONFIG_DEVEL_BOARD=y
-+S:CONFIG_ARMADA_38X_DB=y
 CONFIG_MVEBU_NAND=y
 CONFIG_MVEBU_SPI=y
 CONFIG_MVEBU_PCI=y
 CONFIG_MVEBU_MPP_BUS=y
 +S:CONFIG_MVEBU_I2C=y
++S:CONFIG_DEVEL_BOARD=y
++S:CONFIG_ARMADA_38X_DB=y
 CONFIG_CMD_BDI=y
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
 CONFIG_CMD_EDITENV=y
 CONFIG_CMD_SAVEENV=y
 CONFIG_CMD_MEMORY=y
+CONFIG_CMD_MEMTEST=y
 CONFIG_CMD_LOADB=y
 CONFIG_CMD_LOADS=y
 CONFIG_CMD_FLASH=y
diff --git a/configs/mvebu_armada8k_defconfig b/configs/mvebu_armada8k_defconfig
index f260e48..f300841 100644
--- a/configs/mvebu_armada8k_defconfig
+++ b/configs/mvebu_armada8k_defconfig
@@ -12,6 +12,7 @@ CONFIG_CMD_RUN=y
 CONFIG_CMD_EDITENV=y
 CONFIG_CMD_SAVEENV=y
 CONFIG_CMD_MEMORY=y
+CONFIG_CMD_MEMTEST=y
 CONFIG_CMD_PCI=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_TIME=y
-- 
1.9.1

